wallet/wallet_v4.tact

```diff
@@ -0,0 +1,121 @@
+interface Plugin {
+    internal requested_funds#f06c7567(query_id: Uint64);
+    internal notify_removed#e4737472(query_id: Uint64);
+    internal notify_installed#6e6f7465(query_id: Uint64);
+    internal notify_removed#64737472(query_id: Uint64);
+}
+
+contract Wallet {
+    var seqno: Uint32
+    var subwallet: Uint32
+    var pubkey: Pubkey
+    var plugins: HashSet[Plugin]
+
+    bounced default() {
+        // ignore bounced messages
+    }
+
+    internal default() {
+        // ignore simple transfers
+    }
+
+    internal request_funds#706c7567(
+        query_id: Uint64,
+        requested_amount: Coins,
+        requested_extra: ExtraCurrencies,
+        msg.sender: Plugin,
+    ) {
+        verify(self.balance() - msg.amount >= requested_amount, "Not enough funds");
+
+        send(InboundCoins)
+        msg.sender.requested_funds(
+            query_id,
+            msg.amount: requested_amount,
+            msg.extra: requested_extra,
+        );
+    }
+
+    internal remove_plugin#64737472(query_id: Uint64, msg.sender: Plugin) {
+        self.remove_plugin_from_list(msg.sender);
+
+        if msg.bounceable {
+            send(InboundCoins) msg.sender.notify_removed(query_id);
+        }
+    }
+
+    external pre_condition(body: Slice) -> Slice {
+        let sign = body.read(Signature);
```

Comment:
```suggestion
        let sig = body.read(Signature);
```

"Sign" is usually used for, well, "sign" (as in `+`/`-`). Common abbreviation for signature is "sig".

Reply:
Fixed in [dd0169f](https://github.com/tact-lang/tact-bikeshed/pull/6/commits/dd0169f804887459bfa01020b569aefc36835831)